api/src/main/java/org/motechproject/nms/api/web/BaseController.java

```diff
@@ -24,11 +22,12 @@
     public static final String INVALID = "<%s: Invalid>";
     public static final String NOT_FOUND = "<%s: Not Found>";
 
-    public static final Pattern NUMERIC_PATTERN = Pattern.compile("[1-9][0-9]*|0");
```

Comment:
Do we need to enforce that it starts with a 1?  

Could you just use String regex = "\\d+";